### ISSUE TITLE
Update codebase to Django 2.0 compatibility

### ISSUE THREAD

**Issue by marshalc** [ouh-churchill/quod#7]
Currently breaks when trying to use Django 2.0, needs investigation as to why.

**Comment by marshalc** [comment]
And almost by perfect timing, the 2.0 release of Wagtail appears that should address most of this... https://github.com/wagtail/wagtail/releases (indeed, it apparently was pushed an hour before I logged this ticket) @9711787 - will be worth looking at your requirements files and seeing if the update allows the removal of the "<2" lock on Django

**Comment by marshalc** [comment]
There are some config changes introduced with Wagtail 2.0, so I'll debug those and push the updates later.

**Comment by tomdyson** [comment]
Yes, Wagtail 1.13 didn't support Django 2.0, but Wagtail 2.0 does! Note this is a larger release than usual; you may need to run `wagtail updatemodulepaths`, for example. See the [release notes](http://docs.wagtail.io/en/v2.0/releases/2.0.html#wagtail-module-path-updates) for more details.

**Comment by marshalc** [comment]
Cheers Tom, reading through those at the moment

**Comment by marshalc** [comment]
`TypeError: undefined is not an object (evaluating 'window.draftail.initEditor')` -- current problem stopping RichTextBlocks from being editable. The editor is not appearing/working in the CMS admin.

Changes now in the `wagtail-2-0-upgrade` branch.

So far the site is displaying pages as expected, but can't edit the page content fully in the CMS editor

**Comment by marshalc** [comment]
<img width="820" alt="screen shot 2018-03-01 at 00 22 18" src="https://user-images.githubusercontent.com/572826/36820630-b09b8096-1ce6-11e8-9f56-956b08974a52.png">

Console errors on CMS page edit (e.g. http://localhost:8000/cms/pages/4/edit/)
